rotterdam-nlds-parent-wicket/rotterdam-nlds-wicket/src/main/java/nl/rotterdam/nl_design_system/wicket/components/form_field_label/RdFormFieldLabelBehavior.java

```diff
@@ -0,0 +1,95 @@
+package nl.rotterdam.nl_design_system.wicket.components.form_field_label;
+
+import nl.rotterdam.nl_design_system.wicket.components.base.TagNameClassComponentBehavior;
+import nl.rotterdam.nl_design_system.wicket.components.component_state.Community;
+import nl.rotterdam.nl_design_system.wicket.components.component_state.EstafetteState;
+import nl.rotterdam.nl_design_system.wicket.components.component_state.NlComponentState;
+import nl.rotterdam.nl_design_system.wicket.components.component_state.WicketState;
+import nl.rotterdam.nl_design_system.wicket.components.css_class_names.CssClassNames;
+import org.apache.wicket.Component;
+import org.apache.wicket.markup.ComponentTag;
+import org.apache.wicket.markup.head.CssReferenceHeaderItem;
+import org.apache.wicket.markup.html.form.LabeledWebMarkupContainer;
+import org.jspecify.annotations.Nullable;
+
+import java.util.List;
+
+import static nl.rotterdam.nl_design_system.wicket.components.CssReferenceHeaderItems.cssReferenceHeaderItem;
+import static nl.rotterdam.nl_design_system.wicket.components.form_field_label.RdFormFieldLabelCss.BASE;
+
+/**
+ * <p>
+ * A behavior that turns a <code>&lt;label&gt;</code> into a <a href="https://nldesignsystem.nl/form-field-label/">Form
+ * Field Label</a>.
+ * </p>
+ * <p>
+ * For checkboxes and radio buttons another HTML structure must be used: the input has to be contained in the form field
+ * label.
+ * </p>
+ * <p>
+ * Wicket does not allow both <code>&lt;wicket:id&gt;</code> and <code>&lt;wicket:for&gt;</code> on the same tag. So the
+ * automatic label (based on the label model of the component) and <code>for</code> attribute generated by
+ * <code>&lt;wicket:for&gt;</code> cannot be used. Instead the component the label is for can be set using
+ * {@link #setComponentLabelIsFor(LabeledWebMarkupContainer)}. The component is optional, but should be set in almost
+ * all cases.
+ * </p>
+ * <p>
+ * Based on the <a href="https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-form-label--docs">CSS
+ * implementation of Utrecht</a>.
+ * </p>
+ */
+@NlComponentState(wicketState = WicketState.NEEDS_REFACTORING, estafetteState = EstafetteState.COMMUNITY, htmlCssImplementedBy = Community.UTRECHT)
```

Comment:
Deze is nu een eind goed toch, op UNSTABLE zetten? Dit is de refactoring dat ie goed is toch?